doc/devguide.md

```diff
@@ -105,20 +105,8 @@ Sometimes it is useful to build the Edge Hub container locally. If you want to d
 ## Build Manifest Image
 There is a script in the repo to build multi-architecture images.
 This script assumes that the platform specific images are already in the docker registry.
-Usage is as follows:
-```sh
-$ scripts/linux/buildManifest.sh --help
-
-buildManifest.sh [options]
-Note: Depending on the options you might have to run this as root or sudo.
+Call `scripts/linux/buildManifest.sh --help` for more information.
```

Comment:
Does the buildLocalEdgeHub.sh still work (referenced above)?

Reply:
Honestly, I didn't try it. 😞 I'll give it a shot.

Reply:
@and-rewsmith I just confirmed that `scripts/linux/buildLocalEdgeHub.sh` works as expected. ✅